packages/components/src/components/Input/Input.tsx

```diff
@@ -4,6 +4,7 @@ import { Box } from "../../primitives/Box";
 
 export type InputTypes =
   | "date"
+  | "datetime-local"
```

Comment:
Can you create a story for this type?

Reply:
We don't have stories for all the types, and the design is a browser behavior. That is the reason I haven't created it.